go/proxyd/server.go

```diff
@@ -15,20 +15,27 @@ import (
 	"time"
 )
 
+const (
+	ContextKeyAuth = "authorization"
+)
+
 type Server struct {
-	backends    *BackendGroup
-	maxBodySize int64
-	upgrader    *websocket.Upgrader
-	server      *http.Server
+	backends           *BackendGroup
+	maxBodySize        int64
+	authenticatedPaths map[string]string
```

Comment:
If I understand correctly, `authenticatedPaths` is a mapping of key to user agent?

Reply:
It's key to alias - see above comment.